examples/simple/src/main.rs

```diff
@@ -46,7 +46,7 @@ async fn async_main() {
                 PeerState::Connected => {
                     info!("Peer joined: {:?}", peer);
                     let packet = "hello friend!".as_bytes().to_vec().into_boxed_slice();
-                    socket.send(packet, peer);
+                    socket.channel(0).as_mut().unwrap().send(packet, peer);
```

Comment:
I'm not a fan of how this API changes

Reply:
Not sure why I still have `as_mut` here, it's completely redundant.

Reply:
Could possibly provide a `send_default` method or something on the socket, but I think that actually makes the mental model more ambiguous